Common/src/main/resources/assets/crafttweaker/lang/it_it.json

```diff
@@ -0,0 +1,103 @@
+{
+  "crafttweaker.command.check.log" : "Controlla il file %s!",
+  "crafttweaker.command.click.copy" : "Fai clic per copiare [%s]",
+  "crafttweaker.command.click.goto" : "Fai clic per andare a [%s]",
+  "crafttweaker.command.click.open" : "Fai clic per aprire [%s]",
+  "crafttweaker.command.click.run" : "Fai clic per eseguire [%s]",
+  "crafttweaker.command.conflict.begin" : "Il test sui conflitti per %s è iniziato: ",
+  "crafttweaker.command.conflict.complete" : "Il test sui conflitti è stato completato: i risultati si trovano nel file crafttweaker.log",
+  "crafttweaker.command.conflict.description.output" : " per l'output %s",
+  "crafttweaker.command.conflict.description.type" : " per il tipo %s",
+  "crafttweaker.command.conflict.error" : "Si è verificato un errore durante il test sui conflitti: controlla i log",
+  "crafttweaker.command.conflict.hand.empty" : "Non ci sono oggetti nella mano: impossivile verificare la presenza di conflitti per un oggetto sconosciuto",
+  "crafttweaker.command.conflict.warnings" : "non usare il comando /reload e non chiudere il server durante l'esecuzione",
+  "crafttweaker.command.description.conflicts" : "Identifica e segnala conflitti fra più ricette",
+  "crafttweaker.command.description.ctgui" : "Fornisce informazioni sullo stato di CTGUI",
+  "crafttweaker.command.description.discord" : "Fornisce un link al server Discord di CraftTweaker",
+  "crafttweaker.command.description.docs" : "Apre un link alla documentazione",
+  "crafttweaker.command.description.dump" : "Salva un elenco dei sottocomandi disponibili per il comando /dump",
+  "crafttweaker.command.description.dump.brackets" : "Salva le espressioni tra parentesi uncinate disponibili nella cartella /ct_dumps",
+  "crafttweaker.command.description.dump.info" : "Restituisce un elenco di tutte le espressioni tra parentesi uncinate di tipo '%s' disponibili",
+  "crafttweaker.command.description.dump.loot_modifiers" : "Restituisce i nomi di tutti i modificatori di oggetti registrati",
+  "crafttweaker.command.description.dump.loot_tables" : "Restituisce i nomi di tutte le tabelle di oggetti registrate",
```

Comment:
Uhm, according to the Italian wiki, loot is "rilascio". It would make more sense IMO to use this terminology too. Also because "modificatori di oggetti" is technically wrong...

Reply:
The official translation for "loot table" is "tabella degli oggetti" (see the link I left under the "block entity" discussion), so that preposition should be fixed but otherwise the translation is correct. I translated "loot modifier" as "modificatore di oggetti" both because it has the same function as an [item modifier](https://minecraft.fandom.com/wiki/Item_modifier) ("modificatore di oggetti" in Italian) and by analogy with "tabella degli oggetti". "loot" is usually translated as "oggetti rilasciabili" or "oggetti" in conjunction with the verb "rilasciare". "rilasci" is no longer used in the official Italian translation."

Reply:
I see. Thank you for the clarification